help_to_heat/frontdoor/interface.py

```diff
@@ -244,12 +246,19 @@ def create_referral(self, session_id):
             given_answers[property_type_field] = property_type_field_park_home
             given_answers[property_subtype_field] = property_type_field_park_home
 
-        # override loft access if no loft
+        # Users are given the option to select "no insulation" to improve question usability
+        # This and "below threshold" are functionally identical from the supplier point of view.
+        # They will be combined before the suppliers see it to improve supplier usability
```

Comment:
Swapped out the wrong comment it looks like

Reply:
Ah! Fixed now :)